tests/components/myuplink/test_number.py

```diff
@@ -101,5 +98,5 @@ async def test_entity_registry_smo20(
 ) -> None:
     """Test that the entities are registered in the entity registry."""
 
-    entry = entity_registry.async_get("number.f730_cu_3x400v_change_in_curve")
+    entry = entity_registry.async_get("number.gotham_city_change_in_curve")
```

Comment:
This entity_id change seems weird?

Reply:
https://github.com/home-assistant/core/pull/113534#discussion_r1526602537